docs/source/reference/faq.rst

```diff
@@ -107,6 +107,10 @@ By default, SkyPilot supports most global regions on AWS and only supports the U
   pip install lxml
   # Fetch U.S. regions for GCP
   python -m sky.clouds.service_catalog.data_fetchers.fetch_gcp
+  # Fetch the specified zones for GCP
+  python -m sky.clouds.service_catalog.data_fetchers.fetch_gcp --zones northamerica-northeast1-a us-east1-b us-east1-c
+  # Fetch U.S. zones for GCP, excluding the specified zones
+  python -m sky.clouds.service_catalog.data_fetchers.fetch_gcp --exclude us-east1-a us-east1-b
```

Comment:
I missed the previous PRs. What's the real use case/motivation for excluding some zones?

Reply:
yep, I completed that a couple days ago. more info #2562

Reply:
Checked #2562, didn't seem to find the motivation behind excluding certain zones. Do you mind elaborating?

Reply:
In fact, the `--exclude` flag was added so that when a problem like #2548 is encountered, it can be quickly excluded without modifying the script.
BTW, it is now recommended to use `--zones` to select the specified zones instead of getting all zones, which is more efficient and less likely to cause problems.